proxy-node-acceptance-tests/features/step_definitions/country_journeys.rb

```diff
@@ -0,0 +1,60 @@
+# Start pages
+
+def country_stub_connector_url(country)
+  case country
```

Comment:
How about a map instead of a switch block?

Reply:
Why? The point of a switch is exactly this use case. We don't need to retain the map or anything